src/Razor/test/Microsoft.CodeAnalysis.Razor.Workspaces.Test/Microsoft.CodeAnalysis.Razor.Workspaces.Test.csproj

```diff
@@ -22,6 +22,9 @@
     <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.Workspaces" Version="$(Tooling_MicrosoftCodeAnalysisVisualBasicWorkspacesPackageVersion)" />
     <PackageReference Include="Moq" Version="$(MoqPackageVersion)" />
     <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="$(SystemRuntimeInteropServicesRuntimePackageVersion)" />
+    <EmbeddedResource Include=".\taghelpers.json">
```

Comment:
This looks like its adding a file in this folder, but as a linked file... why? Did you intend to add one of the other existing taghelpers.json as a linked file instead?

Reply:
I'm copying the test data that was in the razor compiler over. If it already exists somewhere in the repo, I'll happily switch.

Reply:
Thats totally fair, but in that case I would expect this to just say `<EmbeddedResource Include="taghelpers.json" />`